dev/reference/system-databases/information-schema.md

```diff
@@ -483,6 +484,16 @@ SHOW TABLES
   [LIKE 'wild']
 ```
 
+Most of the information in the table is the same as MySQL. Only two columns are newly defined by TiDB:
+
+* `TIDB_TABLE_ID`: to indicate the internal ID of a table. This ID is unique in a TiDB cluster.
+* `TIDDB_ROW_ID_SHARDING_INFO`: to indicate the sharding type of a table. The possible values are as follows:
+    - `"NOT_SHARDED"`: the table is not sharded.
+    - `"NOT_SHARDED(PK_IS_HANDLE)"`: the table that defines Primary Key as the row id is not sharded.
+    - `"PK_AUTO_RANDOM_BITS={bit_number}"`: the table that defines Primary Key as its row id is sharded because the Primary Key is assigned with `AUTO_RANDOM` attribute.
```

Comment:
Did you confirm the meaning with @bb7133? If not, please help check.

Reply:
Actually I committed his suggestions for these two lines. You may refer to the conversations above.

Reply:
Got it. I'll take a look.